jitsirtc.js

```diff
@@ -76,19 +76,21 @@ class JitsiRTCClient extends WebRTCInterface {
         auth = {};
       } else {
         // Use custom server config
+        let domainUrl = `${connectionOptions.host}`;
         let mucUrl = `conference.${connectionOptions.host}`;
         let focusUrl = `focus.${connectionOptions.host}`;
         let boshUrl = `//${connectionOptions.host}/http-bind`;
 
         if (game.settings.get('jitsirtc', 'customUrls')) {
+          domainUrl = game.settings.get('jitsirtc', 'domainUrl');
           mucUrl = game.settings.get('jitsirtc', 'mucUrl');
           focusUrl = game.settings.get('jitsirtc', 'focusUrl');
           boshUrl = game.settings.get('jitsirtc', 'boshUrl');
         }
 
         options = {
           hosts: {
-            domain: connectionOptions.host,
+            domain: domainUrl,
```

Comment:
This is the main URL that is being connected to. If you are setting this to meet.jit.si, then you aren't using a custom server you are connecting to Jitsi's public servers.

Reply:
Thanks for looking at this.

the domain: isnt actually what is used to connect but bosh: is.  domain: needs to match the XMPP domain used internally on the prosody setup.  In my example i am setting this to meet.jitsi not the public meet.jit.si.  

https://jitsi.github.io/handbook/docs/devops-guide/devops-guide-docker#advanced-configuration

This link shows the XMPP Domain that needs to match the domain: